tests/models/test_cli.py

```diff
@@ -173,7 +173,7 @@ def test_predict(iris_data, sk_model):
     with TempDir(chdr=True) as tmp:
         with mlflow.start_run() as active_run:
             mlflow.sklearn.log_model(sk_model, "model", registered_model_name="impredicting")
-            model_uri = "runs:/{run_id}/model".format(run_id=active_run.info.run_id)
+            model_uri = f"runs:/{active_run.info.run_id}/model"
```

Comment:
In theory, we can replace all `.format` calls with f-strings, but I'm not a big fan of an expression like:

```python
f"... {some_string.replace('foo', 'bar')} ..."
```